DockerDevSetupGuideForMacOS.md

```diff
@@ -287,19 +287,21 @@ Reference: https://docker-sync.readthedocs.io/en/latest/getting-started/commands
 
 1. Go to http://localhost:3000/ehr-server/reqgen.
 2. Click **Patient Select** button in upper left.
-3. Find **William Oster** in the list of patients and click the dropdown menu next to his name.
-4. Select **E0470** in the dropdown menu.
-5. Click anywhere in the row for William Oster.
-6. Click **Submit** at the bottom of the page.
-7. After several seconds you should receive a response in the form of two **CDS cards**:
-    - **Respiratory Assist Device**
-    - **Positive Airway Pressure Device**
-8. Select **Order Form** on one of those CDS cards.
-9. If you are asked for login credentials, use **alice** for username and **alice** for password.
-10. A webpage should open in a new tab, and after a few seconds, a questionnaire should appear.
-11. Fill out questionnaire and hit next
-<!-- 12. Submit REMS Request to http://localhost:9015/fhir   -->
-13. Submit PAS request to https://davinci-prior-auth.logicahealth.org/fhir
+3. Find **Jon Snow** in the list of patients and click the dropdown menu next to his name.
+4. Select **2183126 - Turalio 200 MG Oral Capsule** in the dropdown menu.
+5. Click anywhere in the row to select Jon Snow.
+6. After several seconds you should receive a response in the form of a **CDS card**:
```

Comment:
are 6 and 8 meant to be the same? I don't recall getting a CDS card after selecting a patient.

Reply:
there are two different card returned, one on order-select and one on order-sign. The order-select one in step 6 doesn't have any links, it just says that this is a REMS Drug